aten/src/ATen/native/native_functions.yaml

```diff
@@ -1105,7 +1105,7 @@
 
 - func: fbgemm_linear_fp16_weight_fp32_activation(Tensor input, Tensor packed_weight, Tensor bias) -> Tensor
 
-- func: fbgemm_pack_quantized_matrix(Tensor input, int K, int N) -> Tensor
+- func: fbgemm_pack_quantized_matrix(Tensor input) -> Tensor
```

Comment:
We need to preserve backward-compatibility here similarly to #22956. I think the procedure should be 

1) Create a new overload without the K and N parameters
2) Move implementation from old function to new function
3) Have the old function call into the new one with a `TORCH_WARN` call warning of deprecation

Reply:
Thanks @jamesr66a ! Updated the code (with 1, 2, and 3) according to your comments.